Lib/fontTools/agl.py

```diff
@@ -1,8 +1,12 @@
+# -*- coding: utf-8 -*-
 # The table below is taken from
 # http://www.adobe.com/devnet/opentype/archives/aglfn.txt
 
-from __future__ import print_function, division, absolute_import
+from __future__ import (print_function, division, absolute_import,
+                        unicode_literals)
```

Comment:
Is `unicode_literals` really necessary here? It breaks some of my doctests under Python 2.7. With this change, `TTFont().getGlyphNames()` can now return a mix of `str` and `unicode`.